app_dart/lib/src/service/scheduler.dart

```diff
@@ -674,6 +699,8 @@ class Scheduler {
   /// Relevant APIs:
   ///   https://developer.github.com/v3/checks/runs/#check-runs-and-requested-actions
   Future<bool> processCheckRun(cocoon_checks.CheckRunEvent checkRunEvent) async {
+    /// Figure out if we're in fusion or not.
```

Comment:
Should this be a TODO? Also, nit: probably not `///`

Reply:
done